models/shopify__products.sql

```diff
@@ -33,7 +69,55 @@ with products as (
         using (order_id, source_relation)
     group by 1,2
 
-), joined as (
+), 
+
+collections_aggregated as (
+
+    select
+        collection_product.product_id,
+        collection_product.source_relation,
+        {{ fivetran_utils.string_agg(field_to_agg='collection.title', delimiter="', '") }} as collections
+    from collection_product 
+    join collection 
+        on collection_product.collection_id = collection.collection_id
+        and collection_product.source_relation = collection.source_relation
+    group by 1,2
+),
+
+tags_aggregated as (
+
+    select 
+        product_id,
+        source_relation,
+        {{ fivetran_utils.string_agg(field_to_agg='value', delimiter="', '") }} as tags
+    
+    from product_tag
+    group by 1,2
+),
+
+variants_aggregated as (
+
+    select 
+        product_id,
+        source_relation,
+        count(variant_id) as count_variants
+
+    from product_variant
+    group by 1,2
```

Comment:
Do you think it would be relevant to include a string agg for the various types of variants? It may not be, but wondering if this would be valuable for customers.

Reply:
yeah the thing is a product can have a TON of variants (i've seen 70 variants for 1 product), so a string_agg could get out of hand

Reply:
Great insight. Let's leave this for now and if someone ever opens a FR for this type of string agg then we can tackle it then.